### DISCUSSION TITLE
change keybindings

### DISCUSSION THREAD

**Discussion by yggdrassilbcn** [seerge/g-helper#2796] [Q&A]
hi everyone. there´s a way to change Ctrl + Shift + F12 - Open G-Helper window to another combination keys? thanks

**Comment by seerge** [comment]
@yggdrassilbcn hello, [there is a way a explained in readme](https://github.com/seerge/g-helper/wiki/Power-user-settings#toggle-performance-mode-or-toggle-app-window-key-binding) :)

**Reply by yggdrassilbcn** [reply]
hi, thanks. i tried, but i didn´t understand about it. 
i want change Ctrl + Shift + F12 - Open G-Helper window to Ctrl + Shift + G - Open G-Helper window

**Reply by seerge** [reply]
@yggdrassilbcn you need to add following line to the beginning of `config.json` (quit app before making changes)

```
"keybind_app": 71,
```

71 is the [code for G](https://www.oreilly.com/library/view/javascript-dhtml/9780596514082/apb.html)

**Reply by yggdrassilbcn** [reply]
thanks a lot. i get it. and thanks for all this work. it´s the best app i can use in my rog ally
